incubator/hnc/hack/post-submit-test.sh

```diff
@@ -0,0 +1,23 @@
+# This file is run by Prow during all postsubmits
+export PATH=$(go env GOPATH)/bin:$PATH
+mkdir -p $(go env GOPATH)/bin
+
+echo "Installing 'kubebuilder' to include the Ginkgo test suite requirements"
+kb=2.3.1
+wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${kb}/kubebuilder_${kb}_linux_amd64.tar.gz
+tar -zxvf kubebuilder_${kb}_linux_amd64.tar.gz
+mv kubebuilder_${kb}_linux_amd64 /usr/local/kubebuilder
+
+# install and setup kind according to https://kind.sigs.k8s.io/docs/user/quick-start/
+echo "Setting up Kind"
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /usr/local/bin/kind
+kind create cluster
+
+hack_dir=$(dirname ${BASH_SOURCE})
+
+echo "Running 'make test-e2e'"
+cd "$hack_dir/.."
+make kind-deploy
+make test-e2e
```

Comment:
Do we need to put a sleep statement in between these two? I thought the e2e tests needed 30s or so for HNC to stabilize.

Reply:
Is it worthwhile to put a check/wait loop in the `make test-e2e` to ensure hnc is deployed and ready?

Reply:
Maybe? It's a bit hard to tell when HNC is ready, and it's never really been an issue before now.

@GinnyJI - is there a pre-suite function we could call to wait for a) the webhooks' caBundles to be set and b) wait until the HNC pod is running? We'd probably want a brief sleep between those two since the pod exits after setting the caBundles and we would want to make sure we only started the wait _after_ that happened.